src/pages/quizzes.tsx

```diff
@@ -0,0 +1,281 @@
+  score: 0,
+  average: 0,
+  completed: JSON.stringify(INITIAL_COMPLETED_QUIZZES),
+}
+
+// Hook
+// TODO: move to other place
+const useLocalStorage = (key, initialValue) => {
+  // State to store our value
+  // Pass initial state function to useState so logic is only executed once
+  const [storedValue, setStoredValue] = useState(() => {
+    if (typeof window === "undefined") {
+      return initialValue
+    }
+    try {
+      // Get from local storage by key
+      const item = window.localStorage.getItem(key)
+      // Parse stored json or if none return initialValue
+      return item ? JSON.parse(item) : initialValue
+    } catch (error) {
+      // If error also return initialValue
+      console.log(error)
+      return initialValue
+    }
+  })
+  // Return a wrapped version of useState's setter function that ...
+  // ... persists the new value to localStorage.
+  const setValue = (value) => {
+    try {
+      // Allow value to be a function so we have same API as useState
+      const valueToStore =
+        value instanceof Function ? value(storedValue) : value
+      // Save state
+      setStoredValue(valueToStore)
+      // Save to local storage
+      if (typeof window !== "undefined") {
+        window.localStorage.setItem(key, JSON.stringify(valueToStore))
+      }
+    } catch (error) {
+      // A more advanced implementation would handle the error case
+      console.log(error)
+    }
+  }
+  return [storedValue, setValue]
+}
+
+const QuizzesHubPage = ({ data }: PageProps<Queries.QuizzesHubPageQuery>) => {
+  const INITIAL_QUIZ = "what-is-ethereum"
+
+  const [currentQuiz, setCurrentQuiz] = useState<string | undefined>(
+    INITIAL_QUIZ
+  )
+  const [nextQuiz, setNextQuiz] = useState<string | undefined>(undefined)
+  const [quizStatus, setQuizStatus] = useState<QuizStatus>("neutral")
+  // TODO: read stats from local storage (useLocalStorage)
+  const [userStats, setUserStats] = useState<UserStats>(INITIAL_USER_STATS)
+  const [isModalOpen, setModalOpen] = useState(false)
+
+  useEffect(() => {
+    // set user stats in Context if available
+    if (localStorage.getItem(USER_STATS_KEY)) {
+      setUserStats(JSON.parse(localStorage.getItem(USER_STATS_KEY)!))
+    } else {
+      // initialize
+      localStorage.setItem(USER_STATS_KEY, JSON.stringify(userStats))
+    }
+  }, [])
+
+  const { t } = useTranslation()
+  const heroContent = {
+    title: <Translation id="quizzes-title" />,
+    header: <Translation id="test-your-knowledge" />,
+    subtitle: <Translation id="quizzes-subtitle" />,
+    image: getImage(data.doge)!,
+    alt: t("quizzes-title"),
+  }
+
+  const contextState = {
+    status: quizStatus,
+    next: nextQuiz,
+    score: userStats.score,
+    average: userStats.average,
+    completed: userStats.completed,
+  }
+
+  return (
+    <Box>
+      <PageMetadata
+        title={t("quizzes-title")}
+        description={t("quizzes-subtitle")}
+      />
+      <HeroContainer>
```

Comment:
Hero H2 should use body color

Reply:
Ignore this, we are using a different version of the hero on Figma that does not exist in the codebase yet. 

It'll update automatically when the new component is implemented.

Reply:
@minimalsm can I mark this as resolved?

Reply:
Yep! This depends on implementing a new hero in design system.